src/ompl/geometric/planners/experience/ert/PlannerData.h

```diff
@@ -0,0 +1,99 @@
+/*********************************************************************
```

Comment:
Is this new PlannerData struct really necessary? From my understanding you are using this to add motions with more than 2 states to the planner data. But you could simply convert the bigger motions to smaller ones (that only have 2 states) and avoid creating a new struct.